src/pages/index.js

```diff
@@ -41,38 +59,77 @@ function handleImageClick(name, link) {
 }
 
 //Popup Forms
-const profilePopup = new PopupWithForm(profileModal, (data) => {
-  console.log(data);
+const profilePopup = new PopupWithForm(profileModal, (data, button) => {
   userInfo.setUserInfo(data);
```

Comment:
You need to update `DOM` only in blocks `then` after a successful response from the server so that when there is an error it wouldn’t change


**Please, correct it everywhere**